cypress/integration/AlertBar/Accepts_icon.feature

```diff
@@ -0,0 +1,13 @@
+Feature: The AlertBars accepts an icon prop
```

Comment:
Perhaps this test could be extended. The internal logic is like this (in order of priority):
- If `icon` is `false`, don't show an icon
- If `icon` is an element, show that element
- Otherwise show a status icon

So I think we should either add a test for the StatusIcon component, or we should check if the correct status-icon is rendered for the corresponding statusProp.

Reply:
Actually, looking at the `AlertBar` tests in general, I think it'd be good to add a section about statuses. Each status has a very different appearance...

Reply:
Yeah that's what I tried to test here. So this: `AlertBar icon shows by default` tests the standard icon, this one ` Scenario: AlertBar icon can be hidden` tests whether it can be disabled with `false` and this one: `Scenario: Custom AlertBar icon` tests whether a custom element will be rendered.

Reply:
> Actually, looking at the AlertBar tests in general, I think it'd be good to add a section about statuses. Each status has a very different appearance...

Yeah, I would also like to test that, but I think visual regression tests would be most appropriate for that. Since that's in my opinion essentially what we want to confirm.

Reply:
Aha gotcha! I think the same will apply to a few of my other comments too then